Source/Tests/TestHarnessRunner.cpp

```diff
@@ -64,6 +64,53 @@ void AssertHandler(char const *Message) {
   fmt::print("[ASSERT] {}\n", Message);
 }
 
+namespace {
+static const std::vector<std::pair<const char*, FEXCore::Config::ConfigOption>> EnvConfigLookup = {{
+#define OPT_BASE(type, group, enum, json, default) {"FEX_" #enum, FEXCore::Config::ConfigOption::CONFIG_##enum},
+#include <FEXCore/Config/ConfigValues.inl>
+}};
+
+// Claims to be a local application config layer
+class TestEnvLoader final : public FEXCore::Config::Layer {
+public:
+  explicit TestEnvLoader(std::vector<std::pair<char const*, char const*>> _Env)
+    : FEXCore::Config::Layer(FEXCore::Config::LayerType::LAYER_LOCAL_APP)
+    , Env {_Env} {
+    Load();
+  }
+
+  void Load() override {
+    std::unordered_map<std::string_view, std::string_view> EnvMap;
+    for (auto &Option : Env) {
+      std::string_view Key = Option.first;
+      std::string_view Value = Option.second;
+
+#define ENVLOADER
+#include <FEXCore/Config/ConfigOptions.inl>
+
+      EnvMap[Key]=Value;
+    }
+
+    std::function GetVar = [=](const std::string_view id)  -> std::optional<std::string_view> {
+      if (EnvMap.find(id) != EnvMap.end())
```

Comment:
```suggestion
      const auto iter = EnvMap.find(id);
      if (iter == EnvMap.end()) {
          return std::nullopt;
      }
      
      return iter->second;
```

or something, so you don't need to hash into the map twice

Reply:
Resolved.